packages/node/src/nodeClient.ts

```diff
@@ -43,39 +66,63 @@ export class NodeClient implements Client<Options> {
     }
 
     this._annotateEvent(event);
+    // Add event to unsent events queue.
+    this._events.push(event);
```

Comment:
There's no mechanism to make sure that events are flushed (or at least attempted to flush) when a server shuts down.

At the bear minimum this should be documented so customers could explicitly call `flush()` and wait but it would be better if we could manage this automatically within the SDK.

I think we could take advantage of the fact that Node will wait for `setTimeout`.